src/unix/fs.c

```diff
@@ -393,8 +395,25 @@ static ssize_t uv__fs_pathmax_size(const char* path) {
 static ssize_t uv__fs_readlink(uv_fs_t* req) {
   ssize_t len;
   char* buf;
+  struct stat st;
+  int ret;
+
+  ret = lstat(req->path, &st);
```

Comment:
This introduces additional I/O on platforms where none is needed (i.e., on platforms where `pathconf(path, _PC_PATH_MAX)` is a constant, like Linux.)

Calling out to lstat() could be acceptable on platforms where there is no maximum path size but it shouldn't impact other platforms.

Reply:
Thanks.  I've updated to use this code path only when a real `PATH_MAX` definition is not available.